ngx_http_uploadprogress_module.c

```diff
@@ -311,18 +311,17 @@ static ngx_str_t ngx_http_uploadprogress_jsonp_multiple_defaults[] = {
 static ngx_array_t ngx_http_uploadprogress_global_templates;
 
 static ngx_str_t*
-get_tracking_id(ngx_http_request_t * r)
+get_tracking_id_internal(ngx_http_request_t *r, ngx_str_t *header_ref, const char *caller, bool multi)
 {
     u_char                          *p, *start_p;
     ngx_uint_t                       i;
     ngx_list_part_t                 *part;
     ngx_table_elt_t                 *header;
     ngx_str_t                       *ret, args;
-    ngx_http_uploadprogress_conf_t  *upcf;
 
-    upcf = ngx_http_get_module_loc_conf(r, ngx_http_uploadprogress_module);
+    (void)multi;
```

Comment:
I fail to understand what this is doing?

Reply:
Also, `multi` is not used anywhere in `get_tracking_id_internal` if I'm not mistaken?

Reply:
Correct. The `(void)varname;` is a common idiom for "this variable is unused". See also my remark about this in my previous comment:

> The `multi` parameter in my implementation is currently unused, but is intended to account for allowing multiple IDs in the header string (and thus possibly multiple instances of the header field).